scripts/plot_ma_key_variables.py

```diff
@@ -2,7 +2,7 @@
 Script to render the moving averages of the variables
 """
 
-from environ.constants import EVENT_DATE_LIST, NETWORK_DATA_PATH
+from data.constants import EVENT_DATE_LIST, NETWORK_DATA_PATH
```

Comment:
why data constants? why don't you just keep the constants in environ?
data folder is typically in .gitignore and wouldn't be pushed 
@lyc0603